render/mapping.go

```diff
@@ -594,6 +594,28 @@ func MapContainer2Pod(n RenderableNode, _ report.Networks) RenderableNodes {
 	return RenderableNodes{id: result}
 }
 
+// MapContainer2Hostname maps container RenderableNodes to 'hostname' renderabled nodes..
+func MapContainer2Hostname(n RenderableNode, _ report.Networks) RenderableNodes {
+	// Propogate all pseudo nodes
+	if n.Pseudo {
+		return RenderableNodes{n.ID: n}
+	}
+
+	// Otherwise, if some some reason the container doesn't have a hostname
+	// (maybe slightly out of sync reports), just drop it
```

Comment:
Yes; if the containers collect gets out of sync for some reason with the endpoint collection, you can end up with a container node with little metadata.  We can safely ignore them in this case.